### DISCUSSION TITLE
Campaign- Video text field not saved

### DISCUSSION THREAD

**Discussion by smsalmanshahAN** [an-nadaaa/nda-front#113] [Feedback]
on the content manager for the campaigns the entry in the "video" field is not saved

When any text is entered in the field "video" and save button is clicked the text from "video "field vanishes

**Comment by khawarizmus** [comment]
I will have to investigate this thank you for reporting it. i will turn my focus to the backend once I "finalize" things on the front end side. currently working on the responsiveness of the campaign page and will jump to finish the causes page

**Comment by khawarizmus** [comment]
Issue created and transferred to the backend repo
